en/components/simple-combo.md

```diff
@@ -0,0 +1,107 @@
+
+## Combo Input Type
+
+The Combo Input provides selection between a border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+
+| Border | Box |
+| ------- | ------- |
+| <img class="responsive-img" src="../images/simple-combo_border.png" srcset="../images/simple-combo_border@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_box.png" srcset="../images/simple-combo_box@2x.png 2x" /> |
+
+## Interaction State
+
+Each of the two Combo Input types can be inserted in an **enabled** or **disabled** state. In Figma we have border and box types available and they can be switched **on** and **off** to a `Disabled` state from the properties panel.
+
+<img class="responsive-img" src="../images/combo_enabledstate.png" srcset="../images/combo_enabledstate@2x.png 2x" />
+<img class="responsive-img" src="../images/combo_disabledstate.png" srcset="../images/combo_disabledstate@2x.png 2x" />
+
+## State
+
+When the user interacts with the Simple Combo, its input goes through various selection states: **Idle** with a placeholder in the place of the content, **Focused** when the input is highlighted, **Filled** once the user has finished with selection and moved on. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
+
+| Idle | Focused | Filled | 
+| ------- | ------- | ------- |
+| <img class="responsive-img" src="../images/simple-combo_idle.png" srcset="../images/simple-combo_idle@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_focused.png" srcset="../images/simple-combo_focused@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_filled.png" srcset="../images/simple-combo_filled@2x.png 2x" /> |
+
+Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Combo Input is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
+
+| Success | Warning | Error | 
+| ------- | ------- | ------- |
+| <img class="responsive-img" src="../images/simple-combo_success.png" srcset="../images/simple-combo_success@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_warning.png" srcset="../images/simple-combo_warning@2x.png 2x" /> | <img class="responsive-img" src="../images/simple-combo_error.png" srcset="../images/simple-combo_error@2x.png 2x" /> |
+
+In Figma changing the states is achieved via the properties panel.
+
+> [!Note]
+> In Figma we have a separate base component called `.Combo Input`, which is explicitly used in the Combo component for both Single and Multiple selection types. The Combo Input has a Prefix container which can be switched **on** and **off**.
+
+## Dropdown Type
```

Comment:
Not sure this is something we should add. I think we need to remove it from the Kit, because this is not consistent at all with everything we have